packages/core/src/request-builder/header-builder/headers-util.ts

```diff
@@ -0,0 +1,71 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
Does it make sense to have tests for the utils methods?

Reply:
yes, will do.

Reply:
Done